Pipfile

```diff
@@ -6,6 +6,7 @@ name = "pypi"
 [packages]
 flask = "*"
 flask-nav = "*"
+fancybox = "*"
```

Comment:
Fancybox for Python is a different thing - it's used for printing message boxes on the commandline, see https://pypi.org/project/fancybox/
```suggestion

```

Reply:
fuk off fancybox why u so confuse